tools/schematics/dt-component-example/index.ts

```diff
@@ -14,37 +14,37 @@
  * limitations under the License.
  */
 
-import { strings } from '@angular-devkit/core';
+import { normalize, strings } from '@angular-devkit/core';
+import { formatFiles } from '@nrwl/workspace';
 import {
   apply,
   chain,
   mergeWith,
+  move,
+  noop,
   Rule,
   template,
-  move,
   Tree,
   url,
-  noop,
-} from '@angular-devkit/schematics';
-import { formatFiles } from '@nrwl/workspace';
-import { DtComponentExampleOptions } from './schema';
-import { normalize } from '@angular-devkit/core';
-import { LICENSE_HEADER } from '../../utils/common-utils';
+  SchematicContext,
+} from '@nrwl/workspace/node_modules/@angular-devkit/schematics';
```

Comment:
never import from node_modules revert this change

Reply:
I removed all sub-node_module imports.